src/offerings.ts

```diff
@@ -325,3 +330,182 @@ export enum PRORATION_MODE {
      */
     IMMEDIATE_AND_CHARGE_FULL_PRICE = 5,
 }
+
+/**
+ * Contains all details associated with a SubscriptionOption
+ * Used only for Google
+ */
+export interface SubscriptionOption {
+    /**
+     * Identifier of the subscription option
+     * If this SubscriptionOption represents a base plan, this will be the basePlanId.
+     * If it represents an offer, it will be {basePlanId}:{offerId}
+     */
+    readonly id: string;
+
+    /**
+     * Identifier of the StoreProduct associated with this SubscriptionOption
+     * This will be {subId}:{basePlanId}
+     */
+    readonly storeProductId: string;
+
+    /**
+     * Identifer of the subscription associated with this SubsriptionOption
+     * This will be {subId}
+     */
+    readonly productId: string;
```

Comment:
I know we've discussed this before, but the naming for `storeProductId` and `productId` is very confusing... Not needed to change this here though.